datastore/client.go

```diff
@@ -50,6 +51,9 @@ func newDatastoreClient(conn grpc.ClientConnInterface, projectID string) pb.Data
 }
 
 func (dc *datastoreClient) Lookup(ctx context.Context, in *pb.LookupRequest, opts ...grpc.CallOption) (res *pb.LookupResponse, err error) {
+	ctx = trace.StartSpan(ctx, "cloud.google.com/go/datastore.Lookup")
```

Comment:
I think the name should be `cloud.google.com/go/datastore.datastoreClient.Lookup` because it's a method on the client. Similar for other methods in this file. @codyoss do you agree?